src/foam/support/view/TicketDetailView.js

```diff
@@ -3,41 +3,119 @@ foam.CLASS({
+        display: inline-block;
+      }
     ^ .foam-u2-UnstyledActionView-backAction {
-      width: 135px;
-      height: 40px;
-      border: 1px solid lightgrey;
-      background-color: rgba(164, 179, 184, 0.1);
-      vertical-align: top;
-      position: sticky;
-      z-index: 10;
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        background-color: rgba(164, 179, 184, 0.1);
+        box-shadow: 0 0 1px 0 rgba(9, 54, 73, 0.8);
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: normal;
+        font-style: normal;
+        font-stretch: normal;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        text-align: center;
+        color: #093649;
     }
+    ^ .Missing-Cash-Out-for {
+        width: 268px;
+        height: 20px;
+        font-family: Roboto;
+        font-size: 20px;
+        font-weight: 300;
+        font-style: normal;
+        font-stretch: normal;
+        line-height: 1;
+        letter-spacing: 0.3px;
+        text-align: left;
+        color: #093649;
+        float:left
+  }
+    ^ .primarydiv{
+      width: 1000px;
+      height: 20px;
+      font-family: Roboto;
+      font-size: 20px;
+      font-weight: 300;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1;
+      letter-spacing: 0.3px;
+      text-align: left;
+      color: #093649; 
+  }
+    ^ .main {
+      width: 488px;
+      height: 16px;
+      opacity: 0.7;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #093649;
+  }
   `,
 
   methods: [
     function initE(){
-      var self = this;
-      
-      console.log(this.data)
-      this.addClass(this.myClass())
-      .start(this.BACK_ACTION).end()
-      // note to ANUBHAV, ticket object is stored in this.data
-      .start().add(this.data.id).end()
-      .start().add(this.data.createdAt).end()
-      .add(this.data.status)
+    var self = this;
+    var email=this.data.supportEmail;
+    this.userDAO.where(this.EQ(this.User.EMAIL, email)).select().then(function(a){
+      self.name=a.a[0].firstName;
+     })
+     var locale = "en-us";
+     var month = this.data.createdAt.toLocaleString(locale, {month: "short"});
+     var date=this.data.createdAt.getDate();
+     var hours=this.data.createdAt.getHours(); 
+     var mins= this.data.createdAt.getMinutes()
+       this.addClass(this.myClass())
+       .start(this.BACK_ACTION).end()
+       .start("br").end()
+       .start("br").end()
+       .start("br").end()
+       .start().addClass('primarydiv')
+            .start().addClass('Missing-Cash-Out-for').add(this.data.subject+"...").end()
+            .start().addClass(this.data.status)
+                .start().add(this.data.status).addClass('Foam-generic-status Ticket-Label-'+ this.data.status).end()
```

Comment:
remove `'Foam-generic-status Ticket-Label-'+ this.data.status` replace with `'Foam-generic-status ' + this.data.status`